test/password-field-validation.html

```diff
@@ -0,0 +1,57 @@
+<!doctype html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>vaadin-password-field validation tests</title>
+
+  <script src="../../web-component-tester/browser.js"></script>
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <link rel="import" href="../vaadin-password-field.html">
+  <link rel="import" href="helpers.html">
+</head>
+
+<body>
+<script>
+  var el = 'vaadin-password-field';
+  describe(`initial validation`, () => {
+    let field, validateSpy;
+
+    beforeEach(() => {
+      field = document.createElement(el);
+      validateSpy = sinon.spy(field, 'validate');
+    });
+
+    afterEach(() => {
+      field.remove();
+    });
+
+    it('should not validate by default', async() => {
+      document.body.appendChild(field);
+      await nextRender();
+      expect(validateSpy.called).to.be.false;
+    });
+
+    it('should not validate when the field has an initial value', async() => {
+      field.value = 'foo@example.com';
```

Comment:
Let's use the value from the original tests.

Reply:
Thanks for fixing it for `password-field`. The comment also applies to the rest components.